lib/modules/session/index.js

```diff
@@ -441,6 +441,18 @@ mage.core.cmd.registerMessageHook('mage.session', false, function (state, cfg, b
 
 
 mage.core.msgServer.msgStream.addAddressType('session', {
+	checkInterval: sessionTTL / 2 * 1000,
+	check: function (address, host, cb) {
```

Comment:
Would calling this `touch` or something not be more appropriate?
When I read `check` I thought it meant the same as `lookup`, but its purpose seems very different.

Reply:
It's a bit tricky since it not only extends but will fail to extend if the session has expired in the meantime. I'll rename it to extend anyway since I agree this makes better sense.